src/service/MessageService.js

```diff
@@ -87,6 +96,11 @@ export async function syncEnvelopes(accountId, folderId, uids, query, init = fal
 	}
 }
 
+export async function prefetchMessage(accountId, folderId, id) {
+	fetchMessage(accountId, folderId, id)
```

Comment:
I think you want to `await` both lines here to limit the nr of concurrent requests a bit